packages.json

```diff
@@ -210,6 +210,7 @@
   "https://github.com/apple/swift-atomics.git",
   "https://github.com/apple/swift-corelibs-xctest.git",
   "https://github.com/apple/swift-crypto.git",
+  "https://github.com/apple/swift-distributed-tracing-baggage-core",
```

Comment:
Wait, I fixed this yesterday - this should have been `...core.git`

Reply:
Need to check if check dependencies wasn't sufficient.

Reply:
Oops, sorry! I didn't spot this with my morning eyes 😂